docs/analyzer-rules/AZFW0013.md

```diff
@@ -0,0 +1,25 @@
+# AZFW0013: Unable to parse binding argument
+
+| | Value |
+|-|-|
+| **Rule ID** |AZFW0013|
+| **Category** |[AzureFunctionsSyntax]|
+| **Severity** |Error|
+
+## Cause
+
+This rule is triggered when a binding attribute argument is an invalid or null value.
+
+## Rule description
+
+[Attributes](https://learn.microsoft.com/en-us/dotnet/csharp/advanced-topics/reflection-and-attributes/) are used to define bindings, and the function metadata generator parses the arguments passed into these attributes to generate binding information during start up.
+
+If the arguments passed in are invalid for any reason (such as being null), this rule is enforced.
```

Comment:
What else would be considered an invalid value?

Reply:
Anything that can't be parsed [here](https://github.com/Azure/azure-functions-dotnet-worker/blob/b4e4ba87f09ce6e7d157d9ed02661bc9db4f4d52/sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs#L700). Shyju and I tried to think through a scenario where this could happen and couldn't come up with one, since you'll get a compiler error before even getting this diagnostic log.